.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
       - id: check-vcs-permalinks
       - id: check-yaml
       - id: debug-statements
-        exclude: '^(raiden/ui/cli.py)|(raiden/tests/conftest.py)$'
+        exclude: '^(raiden/ui/cli.py)|(conftest.py)$'
```

Comment:
Why do we need to exclude these at all?

Reply:
Because pdb is being imported and therefore always leads to false positives on those two files.